concepticondata/conceptrelations.tsv

```diff
@@ -1217,3 +1217,19 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3074	FIX	instanceof	2243	REPAIR
 1755	UNDRESS	instanceof	3538	TAKE OFF
 158	PUT ON	antonymof	3538	TAKE OFF
+3603	EARTH OR LAND	instanceof	1228	LAND
+3603	EARTH OR LAND	instanceof	626	LAND
+3604	LIGHT BLUE	instanceof	837	BLUE
+3604	LIGHT BLUE	narrower	837	BLUE
+3605	LIGHT GREEN	instanceof	1425	GREEN
+3605	LIGHT GREEN	narrower	1425	GREEN
+3606	STURGEON	instanceof	227	FISH
+3606	STURGEON	narrower	227	FISH
```

Comment:
for a kind of fish in comparison to a fish, I suggest not to use the narrower relation. So instanceof is enough.